code/game/machinery/syndicatebomb.dm

```diff
@@ -293,7 +293,7 @@
 	if(adminlog)
 		message_admins(adminlog)
 		log_game(adminlog)
-	explosion(src, range_heavy, range_medium, range_light, flame_range = range_flame)
+	explosion(src, range_heavy, range_medium, range_light, flame_range = range_flame, ignorecap = TRUE)
```

Comment:
uhhh @Jalleo

Reply:
Did you know you can shoot the self-destruct device with a sniper and it drops the bomb core btw? You can then pick it up and blow it up anywhere you want